### DISCUSSION TITLE
HiDPI monitors

### DISCUSSION THREAD

**Discussion by Revdep** [hyprwm/Hyprland#596] [Q&A]
I have a screen with resolution 2560 x 1440 and i have troubles with scaling.
With scaling set to 1 i see very small character and it's hard to read pages, if i set it to 1.8 it's better but the resolution is grainy like a screen 1200x600. 
I though that i miss something. I can't find any hints about it . Thanks for your attention

**Comment by vaxerski** [comment]
not a Hyprland issue. It's an issue with wlroots and XWayland. Only XWayland apps are blurry. There is a workaround, but @fufexan hasnt gotten his ass around to write it down in the wiki yet

**Reply by fufexan** [reply]
I wanna merge my code first, then I'll write all the info in the world.

**Reply by Revdep** [reply]
ok so i just have to wait . Thank you 
and congrats for this awesome compositor

**Reply by 1player** [reply]
Waiting for this info to be written down as well. I gave Sway a try and was saddened to see Xwayland apps are blurry on my 200% scaled screen. I'd love to give hyprland a try if there's a way to have scaled X apps like on GNOME, without having to patch the world.

**Reply by fufexan** [reply]
It's been written for over 3 days already.

**Reply by fufexan** [reply]
https://wiki.hyprland.org/Configuring/XWayland

**Comment by Estebiu** [comment]
I see that it's been added on the wiki; but unfortunately I'm too new to patching&stuff so even reading the arch wiki didn't really help me. I don't know even where to begin. So, would somebody that already has done this send me the commands they've used? I'll be able to figure them out somehow, I just need a starting base. I'm on arch btw. Thank you

**Reply by mfdorst** [reply]
There are links on that page to some guides on how to patch, did you check them out yet? I haven't looked at them so I'm not sure how helpful they are, but you should probably start there.
